frontend/packages/shared/src/types/global.ts

```diff
@@ -29,6 +29,16 @@ export interface ILayoutSettings {
   receiptLayoutName?: string;
 }
 
+export interface ILayoutSets {
+  sets: ILayoutSetConfig[];
+}
+
+export interface ILayoutSetConfig {
+  id: string;
+  dataTypes: string;
+  tasks: string[];
+}
+
```

Comment:
Dette kan godt ligge i en egen fil. På sikt ønsker jeg meg egentlig at vi kvitter oss helt med `globals.ts`. Det er bare en lang liste med typer som ikke henger sammen med hverandre, og som gjør refaktoreringer vanskelige.

`I`-prefikset er vi for øvrig også i ferd med å bevege oss bort fra.